### ISSUE TITLE
Not working in 17.04 ootb

### ISSUE THREAD

**Issue by dakira** [cpriego/valet-linux#71]
Valet version: v2.0.17

This is what I did on a fresh Ubuntu 17.04 install:

    sudo apt install libnss3-tools jq xsel php7.0-cli php7.0-curl php7.0-mbstring php7.0-mcrypt php7.0-xml php7.0-zip php7.0-sqlite3 php7.0-mysql
    composer global require cpriego/valet-linux
    valet install
    cd ~/Sites && valet park

I expected to be able to open something like *project.dev* but that didn't happen. DNS was actually messed up completely (no more resolution at all) until I manually removed all valet configuration files. Running `valet uninstall` did not remove */etc/NetworkManager/dnsmasq.d/default.con* and */etc/NetworkManager/conf.d/valet.conf*. Removing the latter and restarting NetworkManager caused resolution to work again.

The fix in [74cfb2d](https://github.com/cpriego/valet-linux/commit/74cfb2d32c278f71488ab722873d7c8d1b178bb7) does not seem to help.

I tried `valet update` and got multiple errors, one of them a missing `jq` dependency which is not listed anywhere. I'll continue to investigate.

**Comment by dakira** [comment]
*something.dev* resolves to 127.0.53.53 instead of 127.0.0.1 (where nginx is listening).

**Comment by dakira** [comment]
I got it working but I'm not confident it's a good solution for general Ubuntu 17.04 machines. A better solution would be to somehow get everything working with *systemd-resolved*. I don't have enough systemd knowledge for that, though.

Anyway, here is my workaround:

```
sudo systemctl stop systemd-resolved.service
sudo systemctl disable systemd-resolved.service
sudo systemctl stop dnsmasq.service
sudo systemctl disable dnsmasq.service
sudo mv /etc/resolv.conf{,.bak}
sudo ln -s /var/run/NetworkManager/resolv.conf /etc/resolv.conf
sudo systemctl restart NetworkManager.service
```

**Comment by cpriego** [comment]
Fixed on v2.0.18. Does not require workaround.

**Comment by dakira** [comment]
Confirmed.
